pandas/tests/arithmetic/test_numeric.py

```diff
@@ -187,7 +191,8 @@ def test_numeric_arr_rdiv_tdscalar(self, three_days, numeric_idx, box):
         result = three_days / index
         tm.assert_equal(result, expected)
 
-        with pytest.raises(TypeError):
+        msg = "cannot use operands with types dtype"
```

Comment:
This sounds as a very strange error message?

Reply:
I limited the match string to not be too complex.  Here is an example of a full message:
 `ufunc 'multiply' cannot use operands with types dtype('<m8[ns]') and dtype('O')`